### ISSUE TITLE
Issues with the "flat" config exports

### ISSUE THREAD

**Issue by Windvis** [ember-cli/eslint-plugin-ember#2190]
I was trying out the new eslint v9 setup (https://github.com/ember-cli/ember-cli/pull/10516) in our app, but I ran into some issues and one of the issues is related to the "flat" recommended config exports.

Since they are [exported as arrays](https://github.com/ember-cli/eslint-plugin-ember/blob/8e5aaca79330342fe5af0ecbba244d574a3ee5b9/lib/config/recommended.js#L4), they don't work as expected when used as in that PR. The example in the readme also doesn't work as intended.

I've been able to work around the issues by flattening the exported array and then spreading the .rules prop where needed.

(I'll try to add more details later, just creating a new issue before I forget)

**Comment by NullVoxPopuli** [comment]
I think we should add a `import { gjs, gts } from 'eslint-plugin-ember/configs';

where `gjs` is this object:
```js
{
    plugins: { ember: plugin },
    files: ['**/*.gjs'],
    languageOptions: {
      parser: emberEslintParser,
    },
    processor: 'ember/noop',
    rules: gjsRules,
}
```
and `gts` is this object:
```
{
    plugins: { ember: plugin },
    files: ['**/*.gts'],
    languageOptions: {
      parser: emberEslintParser,
    },
    processor: 'ember/noop',
    rules: gtsRules,
}
```

**Comment by NullVoxPopuli** [comment]
I opened #2191 to help out here
